public/stylesheets/style.css

```diff
@@ -58,4 +58,34 @@ body::-webkit-scrollbar {
   
   .colored-toast.swal2-icon-error {
     background-color: #f27474 !important;
+  }
+
```

Comment:
- Here, many CSS attributes seem to be not working, as they have been stacked on top of one another. We could remove many of the irrelevant attributes.